FilePersistence/test/VersionControlMergetests.cpp

```diff
@@ -38,7 +38,7 @@
 #include "ModelBase/src/persistence/PersistentStore.h"
 
 namespace FilePersistence {
-
+/*
```

Comment:
You should not commit any changes that remove old tests. Keep this on your local computer.